osc-control/src/main/java/org/osc/core/server/control/ServerControl.java

```diff
@@ -110,8 +110,8 @@ public boolean isRunning() {
             } else if (cmd.hasOption("check")) {
                 // --check is used by vmidc.sh
                 if (isRunningServer()) {
-                    System.out.println(ServerControl.PRODUCT_NAME + " Server is already running.");
-                    System.out.println(getServerVersion());
+                    log.info(ServerControl.PRODUCT_NAME + " Server is already running.");
```

Comment:
this should be sysout. If the server is running, then server control will not launch and its log is not setup...